site/content/docs/5.3/components/placeholders.md

```diff
@@ -80,7 +80,8 @@ Create placeholders with the `.placeholder` class and a grid column class (e.g.,
 
 We apply additional styling to `.btn`s via `::before` to ensure the `height` is respected. You may extend this pattern for other situations as needed, or add a `&nbsp;` within the element to reflect the height when actual text is rendered in its place.
 
-{{< example >}}
+<!-- Boosted mod: exclude-from-pa11y-analysis because the `a` here is seen as an anchor link -->
+{{< example class="exclude-from-pa11y-analysis">}}
 <p aria-hidden="true">
   <span class="placeholder col-6"></span>
 </p>
```

Comment:
Just saw something weird in the callout message just below (line 93 to 94):

There is a `aria-**disabled**="true"` attribute in the code example and the callout message is explaining the use of `aria-**hidden**="true"`.

It may require some investigation.

Reply:
I think the callout message talks about the line 85 so it might be fine since the `aria-disabled` is on a disabled element.